kmip/core/messages/contents.py

```diff
@@ -118,6 +118,40 @@ def __ne__(self, other):
         else:
             return NotImplemented
 
+    def __lt__(self, other):
+        if isinstance(other, ProtocolVersion):
+            if self.protocol_version_major < other.protocol_version_major:
+                return True
+            elif self.protocol_version_minor < other.protocol_version_minor:
```

Comment:
Oh wow, good catch! I can't believe I missed that. I'll fix this in dependent patch #145.